requirements.txt

```diff
@@ -1,20 +1,20 @@
-numpy==1.21.0
-pandas==1.3.5
-pytorch-lightning==1.5.9
-torch==1.9.0
-torchvision==0.10.0
-gpytorch==1.4.0
-scikit-learn==1.0.2
-tabulate==0.8.9
-pre-commit==2.17.0
-black==22.3.0
-pytest==6.2.5
-setuptools==59.5.0
-flake8==4.0.1
-flake8-bugbear==22.1.11
-gpytorch==1.4.0
-pyreadr==0.4.4
-xlrd==2.0.1
-openpyxl==3.0.9
+numpy>=1.22
+pandas
+pytorch-lightning>=1.6
+torch
+torchvision
+gpytorch
+scikit-learn
+tabulate
+pre-commit
+black
+pytest
+setuptools
+flake8
+flake8-bugbear
+gpytorch
```

Comment:
gpytorch is written twice here (line 6 and line 15)

Reply:
removed